config/liota.conf

```diff
@@ -36,10 +33,10 @@ Apple56 = iotcc_mqtt, iotcc
 Banana23 = iotcc
 
 [DEVSIM_CFG]
-devsim_cmd_msg_pipe = /usr/lib/liota/packages/dev_disc/devsim_cmd_messenger.fifo
+devsim_cmd_msg_pipe = /var/tmp/liota/packages/dev_disc/devsim_cmd_messenger.fifo
```

Comment:
You mentioned to me before the security team needed them to be in the same location as packages for better secuity. So, I don`t think we should remove them from "/usr/lib/liota" as packages and example now exist in this directory.

Reply:
I was told by you that OSS team told you to put liota generated temporary files under /var/tmp folder.
Although security team suggested to put them in the same location, you worked with OSS team and told me above.
I also saw someone (I guess you) modified pkg_msg_pipe back to /var/tmp/liota/package_messenger.fifo
So I just followed the way you did for pkg_msg_pipe. The pipe for device discovery and device simulator should follow the same procedure.
===
[PKG_CFG]
pkg_path = /usr/lib/liota/packages
pkg_msg_pipe = /var/tmp/liota/package_messenger.fifo
pkg_list = /usr/lib/liota/packages/packages_auto.txt

Reply:
The temporary file I previously mentioned were devs,entity and iotcc.json which should exist in "/var/tmp/liota" folder. I never meant the pipe files as you already mentioned before that security team wanted it to be in the same location as the packages or example files. It was modified by John to "/var/tmp/liota/package_messenger.fifo" as there was the disparity in the README which caused him confusion. I think security team will not approve the pipe files in "/var/tmp/liota", if they have suggested they should exist in the same path as packages and examples files.

Reply:
ok. if it is because of misleading communication, will put pipes under /usr/lib